caffe2/proto/caffe2.proto

```diff
@@ -98,10 +98,12 @@ message Argument {
   optional string name = 1;
   optional float f = 2;
   optional int64 i = 3;
-  optional bytes s = 4;
-  repeated float floats = 5;
-  repeated int64 ints = 6;
-  repeated bytes strings = 7;
+  optional string s = 4;
+  optional bytes b = 5;
+  repeated float floats = 6;
+  repeated int64 ints = 7;
+  repeated string strings = 8;
+  repeated bytes bytes = 9;
```

Comment:
It's for backwards compatibility. If someone saves a trained model with one build of Caffe2, they should be able to load it in another build of Caffe2. See, for example, the hoops that the Caffe1 developers were constantly jumping through: https://github.com/BVLC/caffe/blob/rc5/src/caffe/proto/caffe.proto#L10-L22

Now, this only applies if we expect people to be saving to and loading from binary protocol buffers. If that's not an expected use-case, then it doesn't matter. Let's let the @caffe2 guys weigh in on this.